ts/package.json

```diff
@@ -4,9 +4,10 @@
   "description": "",
   "private": true,
   "scripts": {
-    "build": "webpack --mode production",
+    "build": "tsc",
+    "deploy": "webpack --mode production",
```

Comment:
```suggestion
    "compile": "tsc",
    "build": "webpack --mode production",
```

deploy sounds like deploying to a server.

Reply:
Changed to `pack`.